scripts/fix-user.sh

```diff
@@ -0,0 +1,55 @@
+CUID=${1:-$(id -u)}
+CGID=${2:-$(id -g)}
+CUSER=${3:-www-data}
+CGROUP=${4:-www-data}
+# COMMAND_PREFIX may be used to run inside or outside container
+# it can be docker-compose exec -T {containername} or empty string to run locally
+COMMAND_PREFIX=${5:-'docker-compose exec -T php '}
+echo Setting User ${CUSER} = ${CUID} and Group ${CGROUP} = ${CGID}
+
+if [[ $(${COMMAND_PREFIX} grep -c ${CUSER}:x:${CUID}:${CGID} /etc/passwd) == 1 ]]; then
+    echo ${CUSER} already is ${CUID}:${CGID}. No needs to modify users inside container.
+else
+    set -x
+    # Install shadow if it is not exeists as we need usermod and groupmod
+    # For now it is done for Alpine's APK, and not sure is it needed for other OS
+    if [[ (($(${COMMAND_PREFIX} which groupmod) == "") && ($(${COMMAND_PREFIX} which apk) != "")) ]]; then
+        ${COMMAND_PREFIX} apk add --no-cache shadow
```

Comment:
why do you need the package instead of using `sed` & `awk` which is always shipped in every distro?